### PR TITLE
test(e2e): organise ingestion tests

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes:

Fixes <issue-number>

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress.config.ts [hunk 1]
```diff
@@ -29,9 +29,6 @@ export default defineConfig({
     setupNodeEvents(on, config) {
       return plugins(on, config);
     },
-    specPattern: [
-      'cypress/new-tests/**/*.spec.{js,jsx,ts,tsx}',
-      'cypress/e2e/**/*.{js,jsx,ts,tsx}',
-    ],
+    specPattern: ['cypress/e2e/**/*.{js,jsx,ts,tsx}'],
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress.config.ts:32]: dont we need to run test from new-tests folder?

File: openmetadata-ui/src/main/resources/ui/cypress/common/Entities/ServiceBaseClass.ts [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+    this.serviceSetp1(this.serviceType);
+
+    // Enter service name in step 2
+    this.serviceSetp2(this.serviceName);
+
+    // Connection Details in step 3
+    cy.get('[data-testid="add-new-service-container"]')
+      .parent()
+      .parent()
+      .scrollTo('top', {
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/common/Entities/ServiceBaseClass.ts:88]: is it intended to add .parent() twice?

File: openmetadata-ui/src/main/resources/ui/cypress/common/Entities/ServiceBaseClass.ts [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+    );
+    // Navigate to ingested table
+    visitEntityDetailsPage({
+      term: this.entityName,
+      serviceName: this.serviceName,
+      entity: getEntityTypeFromService(this.category),
+    });
+
+    // update description
+    cy.get('[data-testid="edit-description"]').click({ force: true });
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/common/Entities/ServiceBaseClass.ts:350]: force true is not necessary here!

File: openmetadata-ui/src/main/resources/ui/cypress/common/Utils/Services.ts [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+    .invoke('text')
+    .then((text) => {
+      expect(text).to.equal(serviceName);
+    });
+
+  // Clicking on permanent delete radio button and checking the service name
+  cy.get('[data-testid="manage-button"]').click();
+
+  cy.get('[data-menu-id*="delete-button"]').should('be.visible');
+  cy.get('[data-testid="delete-button-title"]').click().as('deleteBtn');
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/cypress/common/Utils/Services.ts:140]:
```suggestion
  cy.get('[data-testid="delete-button-title"]').click();
```
as we are not using it

File: openmetadata-ui/src/main/resources/ui/cypress.config.ts (hunk 1 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/cypress.config.ts:32]: I have merge it with existing structure, it was a temp folder which is removed now

File: openmetadata-ui/src/main/resources/ui/cypress/common/Entities/ServiceBaseClass.ts (hunk 2 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/cypress/common/Entities/ServiceBaseClass.ts:88]: Will check it later

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**